Makefile

```diff
@@ -45,6 +45,7 @@ test: test-unit test-e2e
 .PHONY: test
 
 test-unit:
+    export RELEASE_VERSION ?=0.0.0-0.test-release-version
```

Comment:
This should be fine, but should we use `t.Setenv` in the tests instead? Then we can test what happens when there is no env var, and it will also let you test without using the `make` command.

Reply:
https://pkg.go.dev/testing#B.Setenv